x/oracle/abci.go

```diff
@@ -125,7 +127,10 @@ func Tally(
 			claim := validatorClaimMap[key]
 
 			claim.Weight += tallyVote.Power
-			claim.WinCount++
+			if incrementWin {
+				claim.WinCount++
```

Comment:
Since this is the win count only for mandatory assets now maybe rename it to something like `MandatoryWinCount`?